managed-mode/README.md

```diff
@@ -0,0 +1,25 @@
+# Managed mode
+
+This example shows Buf's [managed mode][managed] in practice. Managed mode is a configuration option that you can set in your [`buf.gen.yaml`][buf-gen-yaml] configuration file. If you set `enabled` to `true`, it instructs the [`buf` CLI][repo] to use an opinionated set of values for each of `protoc`'s natively supported languages, such as Go, Java, and C#. These options are written on the fly into your Protobuf sources so that you don't need to include them in your `.proto` files directly.
+
+To generate code stubs from the Protobuf sources in [`acme/weather/v1`](./acme/weather/v1):
+
+```sh
+buf generate
+```
+
+When you run this command, the `buf` CLI uses the configuration in [`buf.gen.yaml`](./buf.gen.yaml) to generate code stubs. As you can see in that config, [hosted plugins][plugins] are used, [managed mode][managed] is enabled, and several managed mode options are set. See the [`gen/proto`](./gen/proto) directory for generated code in these languages:
+
+* [C++](./gen/proto/cpp)
+* [C#](./gen/proto/csharp)
+* [Go](./gen/proto/go)
+* [Java](./gen/proto/java)
+* [Objective-C](./gen/proto/objc)
+* [PHP](./gen/proto/php)
+* [Ruby](./gen/proto/ruby)
+
```

Comment:
Nit: should this include Swift too? It does have an option baked into the language, but I believe we may not have a hosted plugin for it since its not built into protoc.

https://cs.opensource.google/protobuf/protobuf/+/master:src/google/protobuf/descriptor.proto;l=429-433;drc=1ba1d73e0dcac7c583902c3a757607989515b0bc